src/showcase/app/shared/component-viewer/example-viewer/example-viewer.component.ts

```diff
@@ -43,5 +62,70 @@ export class ExampleViewerComponent implements OnInit {
     this.ts = this._htmlLoader.with(this._route).fromExamples(this.name, 'ts').observe();
 
     this.scss = this._htmlLoader.with(this._route).fromExamples(this.name, 'scss').observe();
+
+    this._htmlLoader
+      .with(this._route)
+      .fromSourceExamples(this.name, 'ts')
+      .observe()
+      .pipe(
+        mergeMap((ts) => {
+          return this._htmlLoader
+            .with(this._route)
+            .fromSourceExamples(this.name, 'html')
+            .observe()
+            .pipe(
+              mergeMap((html) => {
+                return this._htmlLoader
+                  .with(this._route)
+                  .fromSourceExamples(this.name, 'scss')
+                  .observe()
+                  .pipe(
+                    map((scss) => {
```

Comment:
I think you need to check if a scss file really exists. In your current solution, if there is no scss file for an example, a fallback html page is loaded and results in a corrupt stackblitz example. (e.g. https://angular-staging.app.sbb.ch/593/public/components/autocomplete)